python/ray/serve/storage/kv_store.py

```diff
@@ -29,11 +29,19 @@ class RayInternalKVStore(KVStoreBase):
     Supports string keys and bytes values, caller must handle serialization.
     """
 
-    def __init__(self, namespace: str = None):
-        assert ray_kv._internal_kv_initialized()
+    def __init__(
+        self,
+        namespace: str = None,
+        timeout: Optional[float] = None,
```

Comment:
ah can you make this a costants in `ray.serve.constants`

Reply:
@simon-mo  my plan is to add a const in ray.serve.constants in the next PR and pass it to kv_store. Let me just add it here for now.

But I'll give it default value in the next PR (more work for testing and error handling)

Reply:
SGTM